package.json

```diff
@@ -18,6 +18,12 @@
     "test",
     "integration"
   ],
+  "files": [
+    "build/",
+    "typings/index.d.ts",
```

Comment:
Might wanna add `"!**/__tests__"` instead

Reply:
I don't trust globs now, because yarn and npm treats them differently in "files" :<